README.md

```diff
@@ -138,6 +138,39 @@ clustering in the `setup_clustering` recipe:
   of the cluster peer member. Only valid when `node['splunk']['clustering']['mode']='slave'`.
   Defaults to 9887.
 
+The following attributes are related to setting up a Splunk server with search head
+clustering in the `setup_shclustering` recipe:
+
+* `node['splunk']['shclustering']`: A hash of search head clustering configurations
+  used in the `setup_shclustering` recipe
+* `node['splunk']['shclustering']['enabled']`: Whether to enable search head clustering,
+  must be set to `true` to use the `setup_shclustering` recipe. Defaults to `false`,
+  must be a boolean literal `true` or `false`.
+* `node['splunk']['shclustering']['mode']`: The search head clustering mode of the node within
+  the cluster. This is used to determine if the node needs to bootstrap the shcluster and begin
+  captain election. Must be set using string literal 'member' or 'captain'.
+* `node['splunk']['shclustering']['label']`: The label for the shcluster. Used to differentiate 
+  from other shclusters in the environment. Must be a string. Defaults to `shcluster1`.
+  captain election. Must be set using string literal 'member' or 'captain'.
+* `node['splunk']['shclustering']['replication_factor']`: The replication factor
+  of the shcluster. Defaults to 2, must be a positive integer.
+* `node['splunk']['shclustering']['replication_port']`: The replication port
+  of the shcluster members. Defaults to 9900.
+* `node['splunk']['shclustering']['deployer_url']`: The management url for the
+  shcluster deployer server, must be set to a string such as: `https://deployer.domain.tld:8089`.
+  Defaults to empty.
```

Comment:
It's now optional. The `server.conf` template now has a conditional to only write out the `conf_deploy_fetch_url` line if the `deployer_url` attribute is not empty.

I planned on adding deployer configuration later. I'm still kicking around ideas for how to best implement that cleanly. Wanted to get the basic shclustering capability in and functioning in the meantime.